pkg/kots/rego/kots-spec-opa-nonrendered.rego

```diff
@@ -318,6 +318,33 @@ lint[output] {
   }
 }
 
+# Check if the kubernetes installer addons versions are valid
+is_kubernetes_installer(file) {
+  file.content.kind == "Installer"
+  file.content.apiVersion == "kurl.sh/v1beta1"
+} else {
+  file.content.kind == "Installer"
+  file.content.apiVersion == "cluster.kurl.sh/v1beta1"
+}
+is_addon_version_invalid(version) {
+  contains(version, ".x")
+} else {
+  version == "latest"
+}
+lint[output] {
+  file := files[_]
+  is_kubernetes_installer(file)
+  is_addon_version_invalid(file.content.spec[addon].version)
+  output := {
+    "rule": "invalid-kubernetes-installer",
+    "type": "error",
+    "message": "Add-ons versions included in the kubernetes installer must not use 'latest' or the '.x' syntax",
```

Comment:
A suggestion:
>Add-ons included in the Kubernetes installer must pin specific versions rather than "latest" or x-ranges (e.g., 1.2.x).

I just looked it up, and apparently x-range is the terminology for that, but I don't think we've used that term before. ".x patch version" just didn't sound right to me either. Let me know what you think.

Reply:
sounds good, thanks!